src/status_im/notifications/local.cljs

```diff
@@ -95,6 +103,66 @@
      :user-info notification
      :message   description}))
 
+(defn get-chat-type [chat]
```

Comment:
Mmh, do we need to convert to `keyword`? 
I'd use constants for everything, as our definition of `:public?/:group-chat/:one-to-one` falls short to capture all the different kinds of chats and makes refactoring hard otherwise.
You have all you need in `constants.cljs` namespace, and I think we should probably always use those

Reply:
this part is unnecessary, I will just remove it

Reply:
fixed